plugins/mac-virtualcam/src/camera-extension/OBSCameraDeviceSource.swift

```diff
@@ -0,0 +1,288 @@
+//
+//  OBSCameraDeviceSource.swift
+//  mac-camera-extension
```

Comment:
Is "mac-camera-extension" appropriate here, or should this be "camera-extension"? Happy to defer to you.